index.js

```diff
@@ -91,13 +101,15 @@ function checkAndReport(node, timeout) {
 					}
 					console.groupCollapsed('%c%s: %c%s %s', fmt, result.impact, defaultReset, result.help, result.helpUrl);
 					result.nodes.forEach(function (node) {
-						var el = document.querySelector(node.target.toString());
-						console.error(node.failureSummary);
-						console.log('HTML: %c%s', boldCourier, node.html);
-						if (!el) {
-							console.log('Selector: %c%s', boldCourier, node.target.toString());
-						} else {
-							console.log('Element: %o', el);
+						console.error(node.any[0].message);
+						logHtmlAndElement(node);
+
+						if (node.any[0].relatedNodes.length > 0) {
+							console.groupCollapsed('Related nodes');
+							node.any[0].relatedNodes.forEach(function (relatedNode) {
```

Comment:
This is also not sufficient. The above code should show you how to also find all the related nodes for all of the checks that fail the rule.